dbt/adapters/redshift/connections.py

```diff
@@ -253,9 +254,8 @@ def cancel(self, connection: Connection):
             raise
 
         sql = f"select pg_terminate_backend({pid})"
-        _, cursor = self.add_query(sql)
-        res = cursor.fetchone()
-        logger.debug(f"Cancel query '{connection.name}': {res}")
+        cursor = connection.handle.cursor()
+        cursor.execute(sql)
```

Comment:
Debug-level logging for both the high-level message as well as the actual SQL being executed:
```suggestion
        logger.debug(f"Cancel query on: '{connection.name}' with PID: {pid}")
        logger.debug(sql)
        cursor.execute(sql)
```